server/src/main/java/com/netflix/conductor/server/ConductorServer.java

```diff
@@ -57,11 +55,11 @@
 	private static Logger logger = LoggerFactory.getLogger(ConductorServer.class);
 
 	enum DB {
-		redis, dynomite, memory, elasticsearch
+		redis, dynomite, memory, elasticsearch, aurora
 	}
 
 	private enum SearchMode {
-		elasticsearch, memory
+		elasticsearch, memory, none
```

Comment:
Why `none` ?

Reply:
SearchMode specifies the exact index dao backend. elasticsearch - dedicated ES cluster, memory - embedded ES instance (conductor start is), none - our case. For PG implementation, there is no way to specify where the PG can be pointed to - dedicated or embedded like with ES. And also, the `memory` is the default in docker start properties. So, needed something else. We can use 'pg' but in fact that `SearchMode` does not influence anything in case of PG.